xrviz/describe.py

```diff
@@ -38,7 +38,7 @@ def setup(self, var):
         if isinstance(self.data, xr.Dataset):
             self.panel.object = self.variable_pane_for_dataset(var)
         else:
-            self.panel.object = self.variable_pane_for_dataarray(var)
+            self.panel.object = self.variable_pane_for_dataarray()
```

Comment:
Same pattern of many isinstances here.

I'll ask the question again: can we not view a DataArray as a Dataset which happens to have only one variable? It would simplify a lot of this sort of thing.

Reply:
I have thought about this. I don't see a way by which this could be done.

`xr.Dataset` and `xr.DataArray` have to be handled separately. As of now issue of code repeatability is there in dashboard.py, which would be resolved after implemetations of other panels. For most of the functions, the part where these two differ are `data[var]` and `data`.

Reply:
The following is a simple way.

In init of any object taking xarray/dataset:
```
if isinstance(data, xr.DataArray):
    data = xr.Dataset({'data': data})
```

Reply:
or better:
```
data = xr.Dataset({data.name: data}, attrs=data.attrs)
```

Reply:
Thanks a lot sir!!
This is the simple yet the best part of code. It has really simplified everything.